charts/kong/CHANGELOG.md

```diff
@@ -8,6 +8,29 @@ automatically on upgrade. You can fix it by running:
 kubectl apply -f https://raw.githubusercontent.com/Kong/charts/main/charts/kong/crds/custom-resource-definitions.yaml
 ```
 
+## 2.6.2
+
+### Improvements
+
+* Added `app` and `version` labels to pods.
+  ([#504](https://github.com/Kong/charts/pull/504))
+* Reworked leftover socket file cleanup to avoid similar problems of the same
+  class.
+  ([#508](https://github.com/Kong/charts/pull/508))
+
+### Fixed
+
+* SecurityContext and resources applied to PID cleanup initContainer also.
+  ([#503](https://github.com/Kong/charts/pull/503))
+* Disabled the admission webhook on Helm Secrets, fixing an issue where it
+  prevented Helm from updating release metadata.
```

Comment:
```suggestion
* Disabled the admission webhook on Helm Secrets which fixed an issue where it
  prevented Helm from updating release metadata.
```

Reply:
Doing these outside the original changelog update (#509) breaks the normal merge flow for this repo, so leaving this as-is since it's minor.